### PR TITLE
fix: replace roxvet with errcheck

### PR DESCRIPTION
**Pull request by janisz**
## Description

roxvet is in house alternative to errcheck, let's drop it and use analyser that is part of golangci-lint to make use of comunity improvements.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

CI

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

**Review by dhaus67 — APPROVED** [review]

File: .golangci.yml [hunk 1]
```diff
@@ -44,6 +44,25 @@ issues:
...
+    - fmt.Fprint
+    - fmt.Fprintf
+    - fmt.Fprintln
+    - fmt.Print
+    - fmt.Printf
+    - fmt.Println
+    - github.com/stackrox/rox/pkg/utils.Should
+    - io.Copy(*bytes.Buffer)
+    - io.Copy(os.Stdout)
+    - io/ioutil.ReadFile
```
Comment (RTann) [inline .golangci.yml:65]: We already have a rule which disallows `io/ioutil`. Should this be replaced with `os.ReadFile`? Also, when would someone want to ignore the output of this anyway?

Comment (janisz) [inline .golangci.yml:65]:
```suggestion
```

Comment (janisz) [reply .golangci.yml:65]: Right I think we can remove that and io.Copy

**Review by stehessel — APPROVED** [review]

**Pull request merged**